foundation/org.eclipse.persistence.core/src/org/eclipse/persistence/oxm/record/JsonRecord.java

```diff
@@ -12,6 +12,7 @@
 
 // Contributors:
 //     Denise Smith - 2.6 - initial implementation
+//     Radek Felman - 2.7.5 - Bug 389815 - Enhancement Request - JSON specific multidimensional array support
```

Comment:
I think something isn't necessary in the code becuase the credits are in the git commit.

Reply:
This is not about credit, but there are so many changes so just keep there notice.